portfolio/src/main/webapp/script.js

```diff
@@ -79,6 +109,55 @@ class Portfolio {
             alert('Error generated in HTTP response from servlet');
         }
     }
+
+    /**
+    * Adds four maps to the page showing four places I really want to visit.
+    */
+    setupPlaceMaps() {
+        let mapOptions;
+
+        for (let loc of Object.entries(this.visitLocs)) {
+            mapOptions = {  
+                            zoom: 8,
+                            center: loc[1]
+                        };
+            this.createMap(mapOptions, loc[0]);
+        }
+
+    }
+
+    /**
+    * Creates a map with pins of my favorite national parks.
+    */
+    setupParkMap() {
+        let usa = { lat: 44.0733586, lng: -97.5443135 };
+        let mapOptions = {
+            zoom: 3,
+            center: usa
+        };
+        let map = this.createMap(mapOptions, 'parks');
+
+        for (let loc of Object.entries(this.parkLocs)) {
```

Comment:
You can use array destructuring to avoid the need for explicit array indexing:

```javascript
for (const [name, position] of Object.entries(this.parkLocs)) {
```